app/src/scripts/client.jsx

```diff
@@ -2,25 +2,17 @@
 
 import React from 'react'
 import ReactDOM from 'react-dom'
-import Router from 'react-router'
-import routes from './routes.jsx'
-import RouterContainer from './utils/router-container'
-import config from '../../config'
+import {
+  BrowserRouter as Router,
+  Route,
+  browserHistory,
+} from 'react-router-dom'
+import Index from './index.jsx'
+import analyticsWrapper from './utils/analytics.js'
 
-//Google analytics
-import ReactGA from 'react-ga'
-ReactGA.initialize(config.analytics.trackingId)
-
-Router.HistoryLocation.addChangeListener(e => {
-  let path = e.path
-  ReactGA.ga('send', 'pageview', path)
-})
-// intialize router -----------------------------------------------------
-
-let router = Router.create({ routes: routes, location: Router.HistoryLocation })
-
-RouterContainer.set(router)
-
-Router.run(routes, Router.HistoryLocation, function(Handler) {
-  ReactDOM.render(<Handler />, document.getElementById('main'))
-})
+ReactDOM.render(
+  <Router history={browserHistory}>
```

Comment:
is specifying history still needed with react-router v4 (when using BrowserRouter specifically)?

Reply:
Nope, I've taken this out. Thanks!